deployments/charts/kuma/templates/cp-global-sync-service.yaml

```diff
@@ -17,9 +17,9 @@ spec:
   ports:
     - port: {{ .Values.controlPlane.globalZoneSyncService.port }}
       appProtocol: grpc
-  {{- if eq .Values.controlPlane.globalZoneSyncService.type "NodePort" }}
-      nodePort: 30685
-  {{- end }}
+      {{- if and (eq .Values.controlPlane.globalZoneSyncService.type "NodePort") .Values.controlPlane.globalZoneSyncService.nodePort }}
```

Comment:
This is good, so it can be enabled but also the port set to `''` to let k8s pick it